<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DtmfSettingsDialog</class>
 <widget class="QDialog" name="DtmfSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DTMF Settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="labelAlignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>DTMF Transmitting Time</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="txTimeCmbx"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>DTMF Self ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="selfIdTxt">
          <property name="maxLength">
           <number>3</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>DTMF Interval Character</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="intervalCharCmbx"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Group Code</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="groupCodeCmbx"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Decoding Response</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="decodingResponseCmbx"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>First Digit Time[ms]</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="firstDigitTimeCmbx"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Pretime[ms]</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="pretimeCmbx"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Auto Reset Time[s]</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="autoResetTimeCmbx"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Time-Lapse After Encode[ms]</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QComboBox" name="timeLapseCmbx"/>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>PTT ID Pause Time[s]</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QComboBox" name="pttIdPauseCmbx"/>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>D Code Pause</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QComboBox" name="dCodePauseCmbx"/>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Side Tone</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QCheckBox" name="sideToneChbx">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>PTT ID</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QCheckBox" name="pttIdChbx">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>PTT ID Starting(BOT)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pttIdBotTxt">
           <property name="maxLength">
            <number>16</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>PTT ID Ending(EOT)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pttIdEotTxt">
           <property name="maxLength">
            <number>16</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>Remotely Kill</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="remotelyKillTxt">
           <property name="maxLength">
            <number>14</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string>Remotely Stun</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="remotelyStunTxt">
           <property name="maxLength">
            <number>14</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>DTMF Code</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>M1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>M2</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>M3</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>M4</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>M5</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>M6</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>M7</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>M8</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>M9</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>M10</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>M11</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>M12</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>M13</string>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>M14</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>M15</string>
             </property>
            </widget>
           </item>
           <item row="15" column="0">
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>M16</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="m1Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="m2Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="m3Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="m4Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="m5Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="m6Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="m7Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="m8Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLineEdit" name="m10Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="m9Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QLineEdit" name="m11Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QLineEdit" name="m12Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QLineEdit" name="m13Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="QLineEdit" name="m14Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="14" column="1">
            <widget class="QLineEdit" name="m15Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="15" column="1">
            <widget class="QLineEdit" name="m16Txt">
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Allowed Input: 0-9,A,B,C,D,*,#</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="specialCallBtn">
           <property name="text">
            <string>Special Call</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DtmfSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DtmfSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
